blueprints/modular/projects.yaml

```diff
@@ -0,0 +1,42 @@
+title: Projects
+'@extends': default
+
+form:
+  fields:
+    tabs:
+      fields:
+        features:
+          type: tab
+          title: Projects
+          fields:
+            header.projects:
+              name: projects
+              type: list
+              label: Projects

```

Comment:
```suggestion
              label: Projects
              description: Project tiles to be shown after the page's Markdown content.
```

Reply:
I originally had a description on this, but for some reason descriptions dont show up on lists. Can you confirm?

Reply:
You're right. A help tag does show up though - can you do that instead?